frlzjosh/project_management_system
# PMS-20 |  FE - Create Sidebar

## Ready/Hold
`Is this ready to merge?` 
yes
## Description
`What does this branch do?`
Created a sidebar that connects to all pages
## Steps for Testing
`How do we check if this works?`
Pull this branch to make sure that you can see 'deliverables, tasks, etc. '
